### PR TITLE
More Substance Colors

### PR DESCRIPTION
**Pull request by psylucidos**
The current color options are rather limited, it would be a nice addition to allow users to select a broader variety of colors. I added a few more color options for users to assign to different substances.

### CODE REVIEW

File: app/src/main/java/com/isaakhanimann/journal/data/room/experiences/entities/AdaptiveColor.kt [hunk 1]
```diff
@@ -128,7 +128,70 @@ enum class AdaptiveColor {
                 Color(red = 162, green = 132, blue = 94)
             }
         }
+    }
```
Comment (isaakhanimann) [inline app/src/main/java/com/isaakhanimann/journal/data/room/experiences/entities/AdaptiveColor.kt:131]:
```suggestion
    },
```

File: app/src/main/java/com/isaakhanimann/journal/data/room/experiences/entities/AdaptiveColor.kt [hunk 2]
```diff
@@ -128,7 +128,70 @@ enum class AdaptiveColor {
...
+    },
+    INDIGO {
+        override fun getComposeColor(isDarkTheme: Boolean): Color {
+            return if (isDarkTheme) {
+                Color(red = 75, green = 0, blue = 130)
+            } else {
+                Color(red = 74, green = 0, blue = 111)
+            }
+        }
+    },
```
Comment (isaakhanimann) [inline app/src/main/java/com/isaakhanimann/journal/data/room/experiences/entities/AdaptiveColor.kt:176]: There is already a color with the name INDIGO

File: app/src/main/java/com/isaakhanimann/journal/data/room/experiences/entities/AdaptiveColor.kt [hunk 3]
```diff
@@ -128,7 +128,70 @@ enum class AdaptiveColor {
...
+    MAROON {
+        override fun getComposeColor(isDarkTheme: Boolean): Color {
+            return if (isDarkTheme) {
+                Color(red = 110, green = 0, blue = 0)
+            } else {
+                Color(red = 128, green = 0, blue = 0)
```
Comment (isaakhanimann) [inline app/src/main/java/com/isaakhanimann/journal/data/room/experiences/entities/AdaptiveColor.kt:182]: The color for the dark theme should be lighter than the color for the light theme so its better distinguished from the background.
That means that for the dark theme the numbers should be at least as big as for the light theme.

**Review by isaakhanimann — COMMENTED** [review]
Thank you for the suggestion. Please adjust the colors and I will happily merge them and also include them for the iOS app in the next release.

**Comment by psylucidos** [comment]
Thanks for you consideration! I amended the issues in my fork (I must've been under slept), please review the changes if you find the time. On another note thank you for creating this fantastic piece of software, it is incredibly useful.

**Review by isaakhanimann — APPROVED** [review]

**Pull request merged**